encord/user_client.py

```diff
@@ -265,12 +270,30 @@ def create_project(
         dataset_hashes: List[str],
         project_description: str = "",
         ontology_hash: str = "",
-        source_projects: List = list(),
+        workflow_type: ProjectWorkflowType = ProjectWorkflowType.MANUAL_QA,
```

Comment:
Actually, does anyone rely on this `source_projects` already? My main problem here is that you have this extra argument that only sometimes makes sense. I think we can structure the data nicer by only having this option when we have a Manual QA project.
How would you feel about extending the pattern of the CVAT importer with the `ImportMethod`. http://github.com/encord-team/encord-client-python/blob/530950e0be215e02c7bb4a3abc650a59c5352ec5/encord/utilities/client_utilities.py#L58-L70

There, depending on the option you might either have an empty payload, or a payload like the source projects. That way you don't have redundant data.

Reply:
Done exactly that.